client/src/pages/admin/users/UserModal.tsx

```diff
@@ -171,18 +194,17 @@ export default function UserModal({ selectedUserID, onClose }: UserModalProps) {
             </Typography>
 
             <Stack direction="row" spacing={2}>
-              <Button
-                variant="outlined"
-                color="error"
-                startIcon={<DeleteIcon />}
-                onClick={() =>
-                  window.confirm(
-                    "Are you sure you wish to delete John Smith's account? This cannot be undone."
-                  )
-                }
-              >
-                Delete account
-              </Button>
+              {getUserResult.data.user.privilege == Privilege.ADMIN &&
```

Comment:
It's generally better to use `===` instead of `==`, since `==` does weird type conversion stuff.